pymc/tests/distributions/test_mixture.py

```diff
@@ -333,8 +333,8 @@ def test_component_choice_random(self):
             weights = [0.5, 0.5]
             components = [Normal.dist(-10, 0.01), Normal.dist(10, 0.01)]
             mix = Mixture.dist(weights, components)
-        draws = draw(mix, draws=10)
-        # Probability of coming from same component 10 times is 0.5**10
+        draws = draw(mix, draws=200, random_seed=self.get_random_state())
```

Comment:
This should suffice? Trying to keep the tests lean
```suggestion
        draws = draw(mix, draws=20, random_seed=self.get_random_state())
```